pkg/api/domain.go

```diff
@@ -18,6 +17,10 @@ const (
 	ServiceDomainAPPCIRegistry = "registry.ci.openshift.org"
 )
 
+var (
+	ValidClusterNames = sets.NewString(ClusterNameAPPCI, ClusterNameBuild01, ClusterNameBuild02, ClusterNameVSphere)
```

Comment:
Why is this in a different file from the constants?

Reply:
Moved to `constant.go`.